### ISSUE TITLE
Feature Request: Ability to make Subpanels visually indistinct from Module Tabs/Panels

### ISSUE THREAD

**Issue by GeorgeWL** [salesagility/SuiteCRM#2805]
A UI behaviour that a client of mine really likes the idea of, to have all of the necessary data be visible solely through the top tabs.

For example the Detail View data of the Accounts table we have split into Panels, each of these Panels displayed as a Tab [image](http://i.imgur.com/DnpYywc.png) as we really don't like the default mode of Panels rather than Tabs, it makes the user have to scroll far too much and makes the data harder to read in context.

Quick Mockup made through editing in Inspect, this is visual only, not actually real.
[img](http://i.imgur.com/6nipEFb.png)

If I were to take the time, I think I should be able to change this with a piece of custom code relatively easy, but there's other issues that need attending to first.

#### Your Environment
* SuiteCRM Version used: 7.7.6

**Comment by horus68** [comment]
That would be a good feature.
Take a look here: blasher/tabbed_subpanels: A true tabbed subpanel solution for suiteCRM / SugarCRM CE (a custom alternative to the menu-filter-based embedded solution) - https://github.com/blasher/tabbed_subpanels

**Comment by gunnicom** [comment]
Sounds like duplicate to this: https://github.com/salesagility/SuiteCRM/issues/2180

**Comment by GeorgeWL** [comment]
Oops, thought that issue was closed by me

**Comment by GeorgeWL** [comment]
@horus68 not quite, it adds tabs to subpanels, but still keeps them separate below the main section.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Feature Request: Ability to make Subpanels visually indistinct from Module Tabs/Panels](https://trello.com/c/Tg925aH8/45-feature-request-ability-to-make-subpanels-visually-indistinct-from-module-tabs-panels)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
